test/integration/election.test.ts

```diff
@@ -785,4 +786,108 @@ describe('Election integration tests', () => {
       'Could not estimate cost because of negative election blocks size'
     );
   }, 15000);
+  it('should be able to change an election census successfully and perform voting', async () => {
+    const census1 = new PlainCensus();
+    const census2 = new PlainCensus();
+    const voter1 = Wallet.createRandom();
+    const voter2 = Wallet.createRandom();
+    census1.add(await voter1.getAddress());
+    census2.add(await voter1.getAddress());
+    census2.add(await voter2.getAddress());
+
+    const election = createElection(
+      census1,
+      {
+        dynamicCensus: true,
+      },
+      null,
+      census2.participants.length
+    );
+
+    await client.createAccount();
+
+    await client
+      .createElection(election)
+      .then((electionId) => {
+        expect(electionId).toMatch(/^[0-9a-fA-F]{64}$/);
+        client.setElectionId(electionId);
+        return waitForElectionReady(client, electionId);
+      })
+      .then(async () => {
+        // Check voter2 is not in the census
+        const clientVoter2 = new VocdoniSDKClient(clientParams(voter2));
```

Comment:
@marcvelmer future TODO move client1 and client2 instantiation in the intial part of the function. Not sure it is necessary to repeat it in every step.